weather-wise/src/main/java/app/weatherwise/APIService.java

```diff
@@ -15,15 +15,12 @@
 
 public class APIService {
 
-    private final String apiUrl;
-
-    public APIService(String url) {
-        this.apiUrl = url;
+    public APIService() {
```

Comment:
Removed apiUrl as an attribute and added a String url argument to all methods so only one instance of APIService is needed to perform multiple API calls.